openstack/intel-worker/playbooks/intel-worker-playbook.yml

```diff
@@ -1,25 +1,27 @@
 ---
-- name: Create RHCHOS image
+- name: OpenStack worker node configuration Tasks
   hosts: all
-  roles:
-    - rhcosImage-create
+  tasks:
```

Comment:
Ansible warning... unexpected parameter type in action: <class 'ansible.parsing.yaml.objects.AnsibleSequence'>

I'm wondering if we need something changed here.

Reply:
I tried to make some changes here , even in the doc , I did not get what's wrong here , there is no proper explanation why it's wrong .

Reply:
oh! that makes sense to me... thanks for checking